packages/core/src/domain/patient-settings.ts

```diff
@@ -20,6 +20,8 @@ const { log } = out("PatientSettings");
 export type Subscriptions = {
   adt?: string[];
   quest?: boolean;
+  questNotifications?: boolean;
+  questBackfill?: boolean;
```

Comment:
Do we need 3 props for Quest?

Shouldn't we repurpose `quest` to be `questBackfill`, either renaming it or keeping it as is w/ a comment (and leave the refactor for when we ship cohorts)?